turbinia/e2e/run-e2e-gke.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+# Turbinia GKE e2e tests
+# The e2e test will test Turbinia googleclouddisk processing
+# Requirements:
+# - have 'kubectl', 'jq' packages installed
+# - have the Turbinia Helm chart deployed and are authenticated to the GKE cluster
+# - have the 'turbinia-client' CLI installed
+
+set -o posix
+set -e
+
+RELEASE="test"
+DISK="disk-1"
+FAILED=0
+REQUEST_ID=$(uuidgen -rt)
+DATE=$(date -I)
+
+if [ $# -ne  2 ]
+then
+  echo "Not enough arguments supplied, please provide GCP project and zone"
+  echo "$0 [PROJECT] [ZONE]"
+  exit 1
+fi
+
+GCP_PROJECT="$1"
+GCP_ZONE="$2"
+
+echo -n "Started at "
+date -Iseconds
+
+# Back up existing Turbinia config else script will attempt to connect to wrong Turbinia instance
```

Comment:
Ack added a way to restore the backup at the end. holding off on moving things into functions because then I'll want to move everything into functions which could be a nice cleanup but something we can do in like a V2

Reply:
Or we could leave this copy alone and then write the new copy to a different directory (e.g. something like ~/.turbinia-e2e-test/`) and point to it with the command line flag.